invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -156,3 +156,72 @@
   display: inline-block;
   margin-right: 0.5em;
 }
+
+/* Styling for the community selection modal */
+
+#community-selection-modal {
+  margin: 0;
+
+  .community-list-results.content {
+    height: calc(55vh - 10em);
+  }
+
+  .community-list-result-item {
+    .header {
+      white-space: normal;
+      text-overflow: ellipsis;
+      overflow: hidden;
+    }
+  }
+
+  .segment {
+    box-shadow: none;
+  }
+
+  .community-selected-item-container {
+    height: 8rem;
+    min-height: 6rem;
+    overflow-y: hidden;
+    margin: 0;
+    padding: 0.5em 1em;
+
+    .community-list-result-item {
+      height: 7rem;
+    }
+
+    .header {
+      margin: 0;
+    }
+  }
+
+  .community-list-result-item-button-container {
+    margin-top: auto;
+    margin-bottom: auto;
+    width: auto;
+  }
+
+  .community-selection-footer {
+    margin-top: 1.5rem;
+  }
+}
+
+
+@media screen and (max-width: 850px) {
```

Comment:
the breakpoint for mobile phone is diffrent (756px probably, you should check semantic UI configuration for responsive breakpoints)

Reply:
i think for mobile phone it is probably much smaller! This was to include smaller tablets :smile: